jekyll/_cci2/vs-code-extension-quickstart.adoc

```diff
@@ -0,0 +1,117 @@
+---
+contentTags: 
+  platform:
+  - Cloud
+---
+= Get started with the VS Code extension
+:page-layout: classic-docs
+:page-liquid:
+:page-description: A quickstart tutorial for the CircleCI VS Code extension.
+:icons: font
+:toc: macro
+:toc-title:
+
+This tutorial is a quickstart guide to setting up and using the official CircleCI VS Code extension. You can read more about the extension's capabilities in the xref:vs-code-extension-overview#[overview]. 
+
+[#prerequisits]
+== Prerequisites
+
+Before starting this tutorial, make sure you have:
+
+* Installed the Visual Studio Code editor
++
+NOTE: The CircleCI extension is compatible with VS Code version `1.67` or above.
+
+* A CircleCI account (you can link:https://circleci.com/signup/[sign up for free])
+
+* A xref:create-project#[project] you follow that is building on CircleCI
+
+* CircleCI server users: your self-hosted server URL
+
+[#install-the-extension]
+== 1. Install the extension
+
+There are two ways to install the CircleCI extension:
+
+* In VS Code, click the **Extensions** icon on the left-hand Activity Bar, or open **View > Extensions**.
+
+* Or, download the extension from the link:https://marketplace.visualstudio.com/items?itemName=circleci.circleci[VS Code Marketplace].
+
+[#authenticate-into-circleci]
+== 2. Authenticate into CircleCI
+
+. In VS Code, open the CircleCI panel by clicking on the CircleCI icon on the activity bar, then click “Log in”.
+
+. You will be prompted to enter a xref:managing-api-tokens#overview[personal API token]. You may create one in your CircleCI link:https://app.circleci.com/settings/user/tokens[**User Settings > Personal API Tokens**.]
++
+**CircleCI server**: Make sure to check the "I'm a CircleCI Server customer," then enter your self-hosted server URL.
++
+image::{{site.baseurl}}/assets/img/docs/vs_code_extension_login.png[Log in to CircleCI from VS Code]
+
+[#set-up-your-project-in-vs-code]
+== 3. Set up your project in VS Code
+
+NOTE: The pipelines manager does not yet support GitLab projects.
+
+If the root of your workspace is a CircleCI project or contains a project that you follow, your project will be detected automatically, and the pipelines panel will be populated with your most recent pipelines.
+
+Otherwise, you may manually select a project by opening the extension's settings either through the VS Code command `CircleCI: Open Settings`, or by clicking on the settings icon (⚙️) at the top of the pipelines panel.
+
+NOTE: Support for link:https://code.visualstudio.com/docs/editor/multi-root-workspaces[multi-root workspaces] will be available soon.
+
```

Comment:
Is it intentional to have left the image out here? 
No problem if so because you think it's redundant, just want to check

Reply:
@BeFunes Nope, it was a mistake. That said, what do you think of adding the pipelines panel image instead to the "4. Configure the pipelines panel in VS Code" section? I'm thinking after the first paragraph, and before "Specifically, you can choose to display..."

Reply:
I don't feel too strongly about it, but let me explain my reasoning for placing it here
- As soon as your project is setup (detected automatically or selected manually), you see pipelines appear in your pipelines panel. It's the visual confirmation that it's worked, it's the "success" state
- The "configuration" step only serves to filter through which pipelines you want to see. If I were to pick an image for this step, I'd probably show a pic of the settings page for clarity (though we don't want too many pics IMO). 

That said - It's not per se wrong to put the pipelines panel img on the "configure" step, to represent that you can customise it.